rest-tests/src/test/java/io/syndesis/qe/rest/tests/steps/connection/Connections.java

```diff
@@ -34,6 +34,7 @@
     @Autowired
     private ConnectorsEndpoint connectorsEndpoint;
     private final AccountsDirectory accountsDirectory;
+    private Account account;
```

Comment:
I'm a bit torn about this - I understand that we store the account as an instance field to avoid repeating it in the `accountProperty` calls, but it seems kind of hacky.

Reply:
okay :(
![okay :(](https://media3.giphy.com/media/9Y5BbDSkSTiY8/giphy.gif)